### ISSUE TITLE
No Submit button on Android using Firefox

### ISSUE THREAD

**Issue by szabgab** [kantord/LibreLingo#1821]
* Samsung J7 Pro

![no_submit_button](https://user-images.githubusercontent.com/48833/143858861-6764aa43-a15f-4ef6-a272-c7ab7106197c.jpeg)

**Comment by kantord** [comment]
wow. you can consistently reproduce this?

**Comment by szabgab** [comment]
Yes. I can click on "skip" and I get a new question and a skip-button only.

**Comment by szabgab** [comment]
Oh I am stupid. I did not select any of the images and that's why I did not have the submit button. I am not sure why I tried it again and how this time it occurred to me that I need to select an image to see the button.
Feel free to forget about this issue.

Or, maybe it would be better to have a disabled submit button.
